client/src/curriculum/modules-list.tsx

```diff
@@ -29,7 +29,9 @@ export function ModulesListList({
               checked={i === tab}
               onChange={() => setTab(i)}
             />
-            <label htmlFor={`module-${i}`}>{modulesList.title}</label>
+            <label htmlFor={`module-${i}`}>
+              {modulesList.title.replace(/ modules$/, "")}
```

Comment:
Should we have `title` and `short_title`, instead of doing these replacements in the view?

Reply:
No, that's just for the representation in this specific place for now.